packages/backend/vitest.config.integration.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference types="vitest" />
+
+// Configure Vitest (https://vitest.dev/config/)
+// This is for tests that require a database connection.
+
+import { config } from 'dotenv'
+import { resolve } from 'path'
+import { defineConfig } from 'vite'
+
+function getPath(relativePath: string): string {
+  return resolve(__dirname, relativePath)
+}
+
+config({
+  // .env-example should be in the same directory as our config.
+  path: getPath('./.env-example'),
+})
+
+export default defineConfig({
+  test: {
+    name: 'backend-integration',
+    // load env variables
+    setupFiles: ['dotenv/config', getPath('./test/pg-reset-db-setup.ts')],
+    globalSetup: [getPath('./test/pg-global-setup.ts')],
+    include: ['src/**/*.itest.{js,ts}'],
+    threads: false,
```

Comment:
was thinking about it, why shouldn't we run threads? Web requests are concurrent, we want, as much as possible, to be able to catch concurrency problems like forgetting to use transactions, etc.

Reply:
we might run into quite a few issues
- the tables might be truncated while another test is running
- we might be adding a flow for a user while deleting flows for the same user
- checking count of any query might run into race conditions with other tests